java/scripts/docker-checkstyle.sh

```diff
@@ -1,8 +1,42 @@
 #!/bin/bash
 
+help() {
+  echo ""
+  echo "Script to check java code style"
+  echo ""
+  echo "Syntax: "
+  echo ""
+  echo "${SCRIPT} -c /path/to/oscrc"
+  echo ""
+  echo "Where: "
+  echo "  -c  Path to the OSC credentials (usually ~/.osrc)"
```

Comment:
I'd define a define value for this :-)

Reply:
but that could be anywhere....

Reply:
also remove the need for that parameter since the oscrc is now mounted in /root.